### ISSUE TITLE
Error using Unicode characters in Title

### ISSUE THREAD

**Issue by AnasGhrab** [Parisson/Telemeta#156]
I have a fresh install using the docker for testing and I get a 500 error when using unicode characters in the title. For example : "Istiḫbār Raṣd Ḏīl / ʿŪd".

It works on an older install (http://phonotheque.cmam.tn) upgraded to 1.6.3. This is maybe what brings problems with search..

Could it be related to mysql connections ?

**Comment by AnasGhrab** [comment]
Yes, that is certainly the problem. When I do ```SHOW VARIABLES;``` I get : 

```
character_set_client	utf8
character_set_connection	utf8
character_set_database	latin1
character_set_filesystem	binary
character_set_results	utf8
character_set_server	latin1
character_set_system	utf8
character_sets_dir	/usr/share/mysql/charsets/
check_proxy_users	OFF
collation_connection	utf8_general_ci
collation_database	latin1_swedish_ci
collation_server	latin1_swedish_ci
```

On phonotheque.cmam.tn I have the same thing, but as I have created the database manually I'm sure that I've set everything on utf8 : 

```
character_set_client	utf8
character_set_connection	utf8
character_set_database	utf8
character_set_filesystem	binary
character_set_results	utf8
character_set_server	latin1
character_set_system	utf8
character_sets_dir	/usr/local/mysql-5.6.21-osx10.8-x86_64/share/charsets/
collation_connection	utf8_general_ci
collation_database	utf8_general_ci
collation_server	latin1_swedish_ci
```

How does the import acts with that ? Maybe the search problems with arabic are related to that...

**Comment by AnasGhrab** [comment]
The solution should be here : https://hub.docker.com/_/mysql/

It could be done inside my.cnf or : 

```docker run --name some-mysql -e MYSQL_ROOT_PASSWORD=my-secret-pw -d mysql:tag --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci```

But how to include that in the Telemeta docker ?

**Comment by thomasfillon** [comment]
Hi Anas,

Given the two solutions you provide, there is two ways to solve this through Telemeta docker configuration.
For the _custom MySQL configuration file_ approach, as said [here](https://hub.docker.com/_/mysql/) in the section "_Using a custom MySQL configuration file_" : 

> If you want to use a customized MySQL configuration, you can create your alternative configuration file in a directory on the host machine and then mount that directory location as /etc/mysql/conf.d inside the mysql container.

You can get an example of how it can be done [in the Telemeta nginx container configuration](https://github.com/Parisson/Telemeta/blob/master/docker-compose.yml#L84) inside the docker-compose.yml file.

The second solution, _Configuration without a cnf file_, should be easier to achieve through by overriding the Mysql command inside the docker-compose.yml file.
for this, you could try to replace the mysql container configuration by 
In [docker-compose.yml](https://github.com/Parisson/Telemeta/blob/master/docker-compose.yml):
Add the 
```
db:
  image: mysql
  volumes:
    - ./scripts/:/srv/scripts
    - ./data/mysql/:/var/lib/mysql
  volumes_from:
    - data
  env_file:
    - env/prod.env
  command: mysqld  --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci
```
The change is just to add this command line at the end of the mysql container configuration:
`  command: mysqld  --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci`

**Comment by thomasfillon** [comment]
The following question is: How can we convert an existing database in _latin1_ to _utf8_ ?

**Comment by AnasGhrab** [comment]
Thanks thomas, that worked. But I had to remove the containers et do the install procedure again.

I had also exported the database as indicated in the doc, but it didn't work anymore after the backup. The backup gave me the metadata, but there was problem reading the TimeSide img and the audio files. I have to look to the backup procedure closer...

So the question : ```How can we convert an existing database in _latin1_ to _utf8_``` is still not answered, but I close the issue as the main problem is resolved.
